junit-platform-console/src/main/java/org/junit/platform/console/options/Theme.java

```diff
@@ -123,9 +125,8 @@ public String status(TestExecutionResult result) {
 				return aborted();
 			case FAILED:
 				return failed();
-			default:
-				return result.getStatus().name();
 		}
+		return result.getStatus().name();
```

Comment:
I don't like this change.

Having a switch statement for an enum that does not cover all enum constants is an error in my book. Consequently, the code should remain unchanged.